src/store/middlewares/savedAccounts.js

```diff
@@ -0,0 +1,18 @@
+import actionTypes from '../../constants/actions';
+import { successToastDisplayed } from '../../actions/toaster';
+
+const savedAccountsMiddleware = store => next => (action) => {
+  next(action);
+  switch (action.type) {
+    case actionTypes.accountSaved:
+      store.dispatch(successToastDisplayed({ label: 'Account saved' }));
+      break;
```

Comment:
why not use `return` instead of `break` in the switch case? It will look more compact.

Reply:
There's a ticket to fix many other issues about this utility.